### PR TITLE
Docs: Update outdated HTTP Archive links

### LINKED ISSUE #2731: [Docs Bug] `hint-http-cache` outdated HTTP Archive link
The link to HTTP Archive in `hint-http-cache` [#L439](https://github.com/webhintio/hint/blame/master/packages/hint-http-cache/README.md#L439) does no longer return relevant content. I'm unable to find an appropriate replacement in https://httparchive.org/reports.

Perhaps https://mobile.httparchive.org/trends.php#maxage0 or https://desktop.httparchive.org/trends.php#maxage0?

Needless to say, the changes to the HTTP archive site may affect other source links if these exists in other hints documentation.

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #2731

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

**Comment by sarvaje** [comment]
@antross done!

File: packages/hint-image-optimization-cloudinary/README.md [hunk 1]
```diff
@@ -11,9 +11,8 @@ the hint’s configuration. You can create a [free account here][signup].
 
 ## Why is this important?
 
-As of September 15th, 2017, and based on [HTTP Archive][httparchive-report],
-images account for a bit over 53% of the size of a website in average,
-or 1,810kB.
+As of June 1st, 2017, and based on [HTTP Archive][httparchive-report],
```
Comment (antross) [inline packages/hint-image-optimization-cloudinary/README.md]:
```suggestion
As of June 1st, 2019, and based on [HTTP Archive][httparchive-report],
```

Comment (sarvaje) [reply packages/hint-image-optimization-cloudinary/README.md]: 🤦‍♂️

Comment (sarvaje) [reply packages/hint-image-optimization-cloudinary/README.md]: For some reason, I can't apply suggestions... :(

**Pull request merged**